avaframe/data/avaSimilaritySol/Inputs/simiSol_com1DFACfg.ini

```diff
@@ -45,7 +48,7 @@ stopCrit = 0.01
 # friction type (SamosAT = 1, Coulomb = 2)
 frictType = 2
 #+++++++++++++SamosAt friction model
-mu = 0.5773502691896257
+mu = 0.577350269
```

Comment:
why is this not corresponding to delta=21 anymore?

Reply:
there are just less decimals...